example/queue/submit_ncts_intel.job

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+###############################################
+#       Intel MPI job script example          #
+###############################################
+
+#PBS -N gamer                         # Job name
+#PBS -M PUT_YOUR_EMAIL_HERE           # Mail address
+#PBS -m abe                           # Mail notification type (Abort, Begin, End)
+#PBS -l walltime=24:00:00             # Wall time limit (days-hrs:min:sec)
+#PBS -l nodes=gpuserver01:ppn=32
+#PBS -l gres=2                        # Number of requested GPUs per node
+##PBS -o stdout
+##PBS -e stderr
+
+
+if [ "$PBS_ENVIRONMENT" == "PBS_BATCH" ]; then
+   cd $PBS_O_WORKDIR
+fi
+
+
+### Load modules
+module purge
+module load intel-2022.1/compiler
+module load intel-2022.1/openmpi-4.1.2
+module load cuda/11.6
+module load intel-2022.1/fftw-2.1.5_openmpi4
+module load intel-2022.1/hdf5-1.10.8_openmpi4
+
+
+### Specify the GPUs
+#export CUDA_VISIBLE_DEVICES=0,1       # Quadro RTX A6000 (Ampere, sm_86)
+export CUDA_VISIBLE_DEVICES=2,3       # Quadro RTX 6000 (Turing, sm_75)
+
+
+# Using 64 CPUs and 2 GPUs per node, 1 node
+# 2 MPI processes, 32 threads per process (OMP_NTHREAD = 32)
+mpirun -map-by ppr:1:socket:pe=16 --report-bindings ./gamer  1>>log 2>&1
```

Comment:
Doesn't `pe=16` mean using 16 instead of 32 threads per process?

Reply:
@hyschive `pe=n` will bind `n` cores to each process. When using 32 threads (`OMP_NTHREAD = 32`), it means we use the additional threads through hyper-threading. Below are the binding report and performance. The performance are comparable using either 16 or 32 threads.

### 16 threads (OMP_NTHREAD = 16)

```
[gpuserver01:164637] MCW rank 0 bound to socket 0[core 0[hwt 0-1]], socket 0[core 1[hwt 0-1]], socket 0[core 2[hwt 0-1]], socket 0[core 3[hwt 0-1]], socket 0[core 4[hwt 0-1]], socket 0[core 5[hwt 0-1]], socket 0[core 6[hwt 0-1]], socket 0[core 7[hwt 0-1]], socket 0[core 8[hwt 0-1]], socket 0[core 9[hwt 0-1]], socket 0[core 10[hwt 0-1]], socket 0[core 11[hwt 0-1]], socket 0[core 12[hwt 0-1]], socket 0[core 13[hwt 0-1]], socket 0[core 14[hwt 0-1]], socket 0[core 15[hwt 0-1]]: [BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB][../../../../../../../../../../../../../../../..]
[gpuserver01:164637] MCW rank 1 bound to socket 1[core 16[hwt 0-1]], socket 1[core 17[hwt 0-1]], socket 1[core 18[hwt 0-1]], socket 1[core 19[hwt 0-1]], socket 1[core 20[hwt 0-1]], socket 1[core 21[hwt 0-1]], socket 1[core 22[hwt 0-1]], socket 1[core 23[hwt 0-1]], socket 1[core 24[hwt 0-1]], socket 1[core 25[hwt 0-1]], socket 1[core 26[hwt 0-1]], socket 1[core 27[hwt 0-1]], socket 1[core 28[hwt 0-1]], socket 1[core 29[hwt 0-1]], socket 1[core 30[hwt 0-1]], socket 1[core 31[hwt 0-1]]: [../../../../../../../../../../../../../../../..][BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB]


CPU core IDs of all OpenMP threads (tid == thread ID):
------------------------------------------------------------------------
 Rank        Host  NThread  tid-00  tid-01  tid-02  tid-03  tid-04  tid-05  tid-06  tid-07  tid-08  tid-09  tid-10  tid-11  tid-12  tid-13  tid-14  tid-15
    0  gpuserver01       16       9      45      10      46       5      35      47      36       2      43      38      12      32      40      33      39
    1  gpuserver01       16      51      54      49      56      62      16      50      23      57      52      63      29      60      58      21      59


Total Number of Steps : 10
Final Physical Time   : 9.9064407e+00
Total Simulation Time : 79.682825 sec
Timing Diagnosis      :
------------------------------------------------------------------------------------------------------------------------------
                dt  Flu_Adv  Gra_Adv  Src_Adv  Che_Adv       SF    FixUp     Flag   Refine MPI_Grid   Output      Aux       LB CorrSync      Par -MPI_Par    libyt     Init    Other      Sum
    Time      5.29    20.34    28.82     0.00     0.00     0.00     0.26     0.09     0.26     1.76     0.00     1.48     0.02     0.00     0.00     0.00     0.00    20.97     0.38    79.68
    Frac    6.633%  25.523%  36.174%   0.000%   0.000%   0.000%   0.325%   0.119%   0.331%   2.210%   0.000%   1.853%   0.026%   0.000%   0.000%   0.000%   0.000%  26.321%   0.479%  99.993%
```

### 32 threads (OMP_NTHREAD = 32)
```
[gpuserver01:164277] MCW rank 0 bound to socket 0[core 0[hwt 0-1]], socket 0[core 1[hwt 0-1]], socket 0[core 2[hwt 0-1]], socket 0[core 3[hwt 0-1]], socket 0[core 4[hwt 0-1]], socket 0[core 5[hwt 0-1]], socket 0[core 6[hwt 0-1]], socket 0[core 7[hwt 0-1]], socket 0[core 8[hwt 0-1]], socket 0[core 9[hwt 0-1]], socket 0[core 10[hwt 0-1]], socket 0[core 11[hwt 0-1]], socket 0[core 12[hwt 0-1]], socket 0[core 13[hwt 0-1]], socket 0[core 14[hwt 0-1]], socket 0[core 15[hwt 0-1]]: [BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB][../../../../../../../../../../../../../../../..]
[gpuserver01:164277] MCW rank 1 bound to socket 1[core 16[hwt 0-1]], socket 1[core 17[hwt 0-1]], socket 1[core 18[hwt 0-1]], socket 1[core 19[hwt 0-1]], socket 1[core 20[hwt 0-1]], socket 1[core 21[hwt 0-1]], socket 1[core 22[hwt 0-1]], socket 1[core 23[hwt 0-1]], socket 1[core 24[hwt 0-1]], socket 1[core 25[hwt 0-1]], socket 1[core 26[hwt 0-1]], socket 1[core 27[hwt 0-1]], socket 1[core 28[hwt 0-1]], socket 1[core 29[hwt 0-1]], socket 1[core 30[hwt 0-1]], socket 1[core 31[hwt 0-1]]: [../../../../../../../../../../../../../../../..][BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB/BB]


CPU core IDs of all OpenMP threads (tid == thread ID):
------------------------------------------------------------------------
 Rank        Host  NThread  tid-00  tid-01  tid-02  tid-03  tid-04  tid-05  tid-06  tid-07  tid-08  tid-09  tid-10  tid-11  tid-12  tid-13  tid-14  tid-15  tid-16  tid-17  tid-18  tid-19  tid-20  tid-21  tid-22  tid-23  tid-24  tid-25  tid-26  tid-27  tid-28  tid-29  tid-30  tid-31
    0  gpuserver01       32      43       7      47      37       4      44      36      34       3       0      14      13      42      39      46       1       2      12      11      10      35       9      41      45       8      40      32      15       6      38       5      33
    1  gpuserver01       32      31      22      59      17      61      51      52      18      57      26      24      54      19      29      28      27      50      25      56      55      63      23      48      49      20      60      16      62      21      53      30      58


Total Number of Steps : 10
Final Physical Time   : 9.9064407e+00
Total Simulation Time : 76.050849 sec
Timing Diagnosis      :
------------------------------------------------------------------------------------------------------------------------------
                dt  Flu_Adv  Gra_Adv  Src_Adv  Che_Adv       SF    FixUp     Flag   Refine MPI_Grid   Output      Aux       LB CorrSync      Par -MPI_Par    libyt     Init    Other      Sum
    Time      5.42    20.58    27.27     0.00     0.00     0.00     0.26     0.12     0.38     2.54     0.00     2.21     0.01     0.00     0.00     0.00     0.00    16.88     0.38    76.04
    Frac    7.132%  27.057%  35.852%   0.000%   0.000%   0.000%   0.340%   0.154%   0.502%   3.343%   0.000%   2.903%   0.016%   0.000%   0.000%   0.000%   0.000%  22.192%   0.500%  99.992%
```

Reply:
@hfhsieh Thanks for the confirmation!